### PR TITLE
Add tags mapping to organisation generic counters

### PR DESCRIPTION
**Pull request by RaduW**
This PR adds mappings for accessing tags in organisation generic counters

### CODE REVIEW

**Review by onewland — COMMENTED** [review]
Do we want to support tags on `org_counters`? This is generally for reporting functions (cross-org querying) and I would rather we don't expose this "heavy" column if it's not in use.

**Comment by RaduW** [comment]
Hello @onewland the reason I need the tags exposed in the org_counters is that I need transaction names for a recurring task that rebalances transaction sampling rates for all projects. 

So basically a recurrent job that runs every 6 hours or so and based on the relative frequencies of transactions whithin a project it boosts infrequent transactions and pushes down frequent transactions in order to insure that all transaction types have an appropriate amount of instances kept by our sampling algorithm. 

This means that I need to go over the counters of all transactions on all projects of all organisations that were active in the last interval. 

I realise that this is an immense number of rows retrieved from the database, I am running the query in batches of 1000 rows. Please advise if you have any concerns  or if there are better ways of going about what I'm doing.

File: snuba/datasets/configuration/generic_metrics/entities/org_counters.yaml [hunk 1]
```diff
@@ -8,6 +8,18 @@ schema:
     { name: metric_id, type: UInt, args: { size: 64 } },
     { name: timestamp, type: DateTime },
     { name: bucketed_time, type: DateTime },
+    {
```
[the remark this answers is not in the document]
Comment (RaduW) [reply snuba/datasets/configuration/generic_metrics/entities/org_counters.yaml:11]: Added tags also to storages, but I don't really know what I'm doing though.

**Review by evanh — APPROVED** [review]
Pending CI passing, this looks good to me.

**Pull request merged**